projects/js-packages/connection/state/actions.jsx

```diff
@@ -46,6 +54,14 @@ const fetchAuthorizationUrl = redirectUri => {
 	return { type: FETCH_AUTHORIZATION_URL, redirectUri };
 };
 
+const fetchConnectedPlugins = () => {
+	return { type: FETCH_CONNECTED_PLUGINS };
+};
```

Comment:
Looks like this isn't being used, or I missed some place?

Reply:
you're right. Removed in https://github.com/Automattic/jetpack/pull/23358/commits/afc723407612a2d5231be3b085b91a4c8888c709